configuration.php

```diff
@@ -4,3 +4,9 @@
 
 $this->menuSection(N_('Graphite'), ['icon' => 'chart-area'])->setUrl('graphite/show/overview');
 
+$this->provideConfigTab('backend', array(
+    'title' => $this->translate('Configure the Graphite Web backend'),
+    'label' => $this->translate('Backend'),
```

Comment:
Would be better to change this to 'Configuration', so it's clear what you can expect on this page.

Reply:
1. The only way to the page is **Configuration** > Modules > Graphite (not clear enough?)
2. The monitoring module does it the same way

Reply:
You got me with "2." 👍